cli/tests/upgrade.bats

```diff
@@ -173,106 +175,92 @@ EOF
 # catalog tests
 
 @test "catalog: upgrade hello" {
-  skip "FIXME: upgrades package but reports that no packages were upgraded"
-
   "$FLOX_BIN" init
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/old_hello.json" "$FLOX_BIN" install hello
 
-  old_hello_response_rev=$(jq -r '.[0].[0].pages.[0].page' "$GENERATED_DATA/resolve/old_hello.json")
-  old_hello_locked_rev=$(jq -r '.packages.[0].rev_count' "$LOCK_PATH")
-  assert_equal "$old_hello_locked_rev" "$old_hello_response_rev" 
+  old_hello_response_drv=$(jq -r '.[0].[0].page.packages[0].derivation' "$GENERATED_DATA/resolve/old_hello.json")
+  old_hello_locked_drv=$(jq -r '.packages.[0].derivation' "$LOCK_PATH")
+  assert_equal "$old_hello_locked_drv" "$old_hello_response_drv" 
 
   _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
     run "$FLOX_BIN" upgrade
   assert_output --partial "Upgraded 'hello'"
```

Comment:
Asserting on the full output here (and the related tests) would have caught the duplication issue without relying on the other bugs:
```suggestion
  assert_output  "⬆️  Upgraded 'hello' in environment '$PROJECT_NAME'."
```

Reply:
That's a good point, but for now I'm just straight up copying the tests and making them work for the sake of time.